### ISSUE TITLE
noise_meter 3.0.3

### ISSUE THREAD

**Issue by ManthanSutariya** [cph-cachet/flutter-plugins#509]
### Device / Emulator and OS
 - Device: iPhone 13
 - OS: iOS 15.2

### Describe the bug
[!] No podspec found for `audio_streamer` in `.symlinks/plugins/audio_streamer/ios`

### Run Pod install inside ios directory
pod install

**Comment by dannyalbuquerque** [comment]
+1

**Comment by selarom-epilef** [comment]
I'm also having the same issue. Is there any workaround in the meantime?

**Comment by devibrahimkarahan** [comment]
You should copy these example project ios folder content to your folder.

Example project > ios > .symlinks > plugins > audio_streamer > ios > [Copy all folder and files]

Paste it your project > ios > .symlinks > plugins > audio_streamer > ios > Paste here

And now you can install pod, it worked on me

**Comment by asawilliams** [comment]
+1

**Comment by dfdgsdfg** [comment]
## Reason - #540 
audio_streamer deployed not propely on pub.dev.

## Work around
Add below your pubspec.yaml. It is update your `audio_streamer` from the latest git not pub.dev.

```yaml
dependency_overrides:
  audio_streamer:
    git:
      url: https://github.com/cph-cachet/flutter-plugins.git
      path: packages/audio_streamer
      ref: master
```

**Comment by bardram** [comment]
Closed with #540
